example/build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "play-googleauth-example"
 
-version := "0.3.5-SNAPSHOT"
+version := "0.3.8-SNAPSHOT"
```

Comment:
I agree, we should bump it to 0.4, as bumping Play to 2.5.x is a breaking change.

The usual way of handling versioning in cases like this is to increment your library's version whenever you increment the corresponding Play version. e.g. 0.3.x is compatible with Play 2.4.x, 0.4.x is compatible with Play 2.5.x, etc. If we ever need to release a fix for the Play 2.4.x version of the lib, we can create a `0.3.x` branch from the `v0.3.7` tag.

I'd also add a note to the readme explaining this versioning policy.